src/components/LandingSection/LandingSection.js

```diff
@@ -15,17 +17,48 @@ const LandingHeader = styled.h1`
   font-family: ${fonts.families.headerFont};
   font-weight: ${fonts.weights.regular};
   width: 80%;
-  word-break: break-all;
+`;
+
+const ChangeLocaleWrapper = styled.div`
+  position: fixed;
+  top: ${spacing.xlSize};
+  left: -${spacing.xxxxlSize};
+
+  transform: ${({ isNavActive }) =>
+    isNavActive ? 'none' : 'translateX(120px)'};
+  transition: transform 0.5s ease-in-out;
+`;
+
+const ChangeLocaleButton = styled.button`
+  margin-right: ${spacing.mSize};
+  background: inherit;
+  border: none;
+  cursor: pointer;
+
+  &:focus {
```

Comment:
you have to style somehow focused button if you are using `outline: none` due to a11y